sunpy/database/tests/test_database.py

```diff
@@ -625,6 +627,7 @@ def test_add_entries_from_qr_ignore_duplicates(database, query_result):
     assert len(database) == 32
 
 
+@pytest.mark.skip
```

Comment:
I forgot if there was a reason for this?

Reply:
This is just the NOAA servers melting, I will revert the commit.